Blish HUD/Controls/Intern/Keyboard.cs

```diff
@@ -106,15 +77,15 @@ public void Release(GuildWarsControls key)
             }
             else
             {
-                uint vkCode = (uint)VirtualKeyShorts[key];
+                uint vkCode = (uint)key;
                 ExtraKeyInfo lParam = new ExtraKeyInfo
                 {
                     scanCode = (char)PInvoke.MapVirtualKey(vkCode, MAPVK_VK_TO_VSC),
                     repeatCount = 1,
                     prevKeyState = 1,
                     transitionState = 1
                 };
-                PInvoke.PostMessage(Gw2WndHandle, WM_KEYUP, vkCode, lParam.GetInt());
+                PInvoke.PostMessage(GameService.GameIntegration.Gw2Process.MainWindowHandle, WM_KEYUP, vkCode, lParam.GetInt());
```

Comment:
Use `GameService.GameIntegration.Gw2WindowHandle` instead of pulling it from the `Gw2Process` since it is already exposed.